axon-server-connector/.factorypath

```diff
@@ -0,0 +1,65 @@
+<factorypath>
```

Comment:
I think this file was unintentionally added. Would you mind removing it from the PR?

Reply:
Yes.  I removed it. 
I've also added annotation and test to some more serializable objects (had overseen them)
Unit-Tests ran through this time.